src/components/ContactCard/ContactGroups.jsx

```diff
@@ -32,11 +32,18 @@ export class ContactGroupsClass extends React.Component {
     const { contact, allGroups } = this.props
 
     if (isGroupAlreadyExist(allGroups, group)) {
-      return
+      return Alerter.error(
+        this.props.t('groups.already_exists', { name: group.name })
+      )
     }
+
     const createdGroup = await this.props.createGroup(group)
+    if (get(createdGroup, 'data.name') === group.name) {
+      await contact.groups.addById(createdGroup.data._id)
+      return Alerter.success(this.props.t('groups.created.success'))
+    }
```

Comment:
Same. Try catch seems better, no ?

We can also group the both async call in the try / catch to display an error also if the addById has failed